pkg/flow/adapter/xmltojsontransformation/adapter_test.go

```diff
@@ -0,0 +1,136 @@
+/*
+Copyright 2022 TriggerMesh Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package xmltojsontransformation
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	cloudevents "github.com/cloudevents/sdk-go/v2"
+	cetest "github.com/cloudevents/sdk-go/v2/client/test"
+	"github.com/stretchr/testify/assert"
+
+	"knative.dev/eventing/pkg/adapter/v2"
+	logtesting "knative.dev/pkg/logging/testing"
+)
+
+const (
+	tBridgeID         = "bride-abdc-0123"
+	tCloudEventID     = "ce-abcd-0123"
+	tCloudEventType   = "ce.test.type"
+	tCloudEventSource = "ce.test.source"
+	tSuccessAttribute = "success"
+	tErrorAttribute   = "error"
```

Comment:
The two constants above are not used (I'm surprised the linter didn't find this).